translations/monero-core_es.ts

```diff
@@ -1085,235 +1085,243 @@ The following information will be deleted
 
 The old wallet cache file will be renamed and can be restored later.
 </source>
-        <translation type="unfinished"></translation>
+        <translation>¿Seguro que deseas reconstruir el caché del monedero?
+La siguiente información será eliminada
+- Direcciones de destinatarios
+- Claves de transacciones
+- Descripciones de transacciones
+
+El archivo de caché anterior del monedero será renombrado y puede ser posteriormente restaurado.</translation>
     </message>
     <message>
         <location filename="../pages/settings/SettingsInfo.qml" line="188"/>
         <source>Cancel</source>
-        <translation type="unfinished">Cancelar</translation>
+        <translation>Cancelar</translation>
     </message>
     <message>
         <location filename="../pages/settings/SettingsInfo.qml" line="202"/>
         <source>Invalid restore height specified. Must be a number.</source>
-        <translation type="unfinished"></translation>
+        <translation>Altura de restaurado especificada inválida. Debe ser un número.</translation>
     </message>
     <message>
         <location filename="../pages/settings/SettingsInfo.qml" line="236"/>
         <source>Wallet log path: </source>
-        <translation type="unfinished"></translation>
+        <translation>Dirección de registro del monedero:</translation>
```

Comment:
Missing another space oups...

Reply:
Fixing this and pushing again, thanks

Reply:
Found some other missing spaces with the Search tool looking at ':<', and changed just the ones that are missing. Left the other ones that are pair (the source text doesn't include a space) intact.